src/libs/SidebarUtils.ts

```diff
@@ -360,9 +360,9 @@ function getOptionData({
     const status = personalDetail?.status ?? '';
     const formattedLogin = Str.isSMSLogin(login) ? LocalePhoneNumber.formatPhoneNumber(login) : login;
 
-    // We only create tooltips for the first 10 users or so since some reports have hundreds of users, causing performance to degrade.
+    // We only create tooltips for the first 5 users or so since some reports have hundreds of users, causing performance to degrade.
     const displayNamesWithTooltips = ReportUtils.getDisplayNamesWithTooltips(
-        (participantPersonalDetailList || []).slice(0, 10),
+        (participantPersonalDetailList || []).slice(0, 5),
```

Comment:
Can you explain how this can fix the inconsistent in name while the inconsistent is at the name beginning not at the end?

For example the current name is `name3, name4, name5, name6, ...`. how `.slice(0, 5)` can fix it to `name1, name2, name3, name4, name5, ...`?

Reply:
The issue is not fixed

![Screenshot 2024-09-04 at 1 36 50 PM](https://github.com/user-attachments/assets/1276c127-d389-4529-ab84-695096167d35)

Reply:
Note `getDisplayNamesWithTooltips` use sort method but get `getReportName` is not

https://github.com/Expensify/App/blob/0f26b8a70b92f79804b7a31cbe8e0bf027d772cb/src/libs/ReportUtils.ts#L2387-L2396

Reply:
> Can you explain how this can fix the inconsistent in name while the inconsistent is at the name beginning not at the end?

Since `getDisplayNamesWithTooltips` sorts the array by name, it's possible the account in index 5-9 has a "smaller" name.

Slicing it to 5 will make sure the account list will be the same between `getDisplayNamesWithTooltips` and `getReportName`.

> Note getDisplayNamesWithTooltips use sort method but get getReportName is not
> It looks LHN and header use getDisplayNamesWithTooltips and details page use getReportName

True, `getDisplayNamesWithTooltips` is used in LHN and header so there is a tooltip for each account.
Should we sort in getReportName too? Or should we remove the sort from `getDisplayNamesWithTooltips`?

Reply:
I think we should sort the both, but I prefer fixing it separately here https://github.com/Expensify/App/issues/47803. I just wanted to confirm it is not a regression from our PR